recipes/oommf/bld.bat

```diff
@@ -0,0 +1,5 @@
+curl -o oommf.zip  "http://math.nist.gov/oommf/dist/oommf12b0_20160930_86_x64.zip"
```

Comment:
This is downloaded and extracted here because of https://github.com/conda-forge/staged-recipes/pull/3038/files#diff-e41816a448c34eb6ab6f515c151653a2R10. You don't need to download it again here.